guides/common/modules/con_managing-dns-using-smart-proxies.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Managing_DNS_Using_Smart_Proxies_{context}"]
+= Managing DNS Using {SmartProxies}
+
+{Project} can manage DNS records using your {SmartProxy}.
+DNS management contains updating and removing DNS records from existing DNS zones.
+Your {SmartProxy} can use multiple providers, allowing you to integrate with your existing DNS infrastructure or deploy a new one.
+
+After you have enabled DNS, your {SmartProxy} can manipulate any DNS server that complies with RFC 2136.
```

Comment:
This is only with the default provider.

background: [nsupdate](https://en.wikipedia.org/wiki/Nsupdate) is the tool used to implement the client side of [RFC 2136](https://datatracker.ietf.org/doc/html/rfc2136). That's why the provider is named `dns_nsupdate`.

Probably not the ideal text, but have you considered something like:
```suggestion
After you have enabled DNS, your {SmartProxy} can manipulate any DNS server that complies with RFC 2136 using the `dns_nsupdate` provider.
Other providers provide more direct integration, such as `dns_infoblox` for [Infoblox](https://www.infoblox.com/).
```

This would tie in to the last sentence of the previous paragraph.

Reply:
Thanks, applied!